first_hour_on_vwb/working_with_resources.ipynb

```diff
@@ -855,58 +855,38 @@
     "        self.input_name = wu.TextInputWidget(\"<NAME>\", \"Name:\").get()\n",
     "        self.output = widgets.Output()\n",
     "        self.button = wu.StyledButton('Create dataset','Click to create a BigQuery dataset as a controlled resource.','plus').get()\n",
-    "        self.vb = widgets.VBox([\n",
-    "            self.label,\n",
-    "            self.warning,\n",
-    "            self.access_drop_down,\n",
-    "            self.cloning_drop_down,\n",
-    "            self.input_dataset_id,\n",
-    "            self.input_description,\n",
-    "            self.input_name,\n",
-    "            self.button,\n",
-    "            self.output\n",
-    "        ], layout=wu.vbox_layout)\n",
+    "        self.toggle_optional = widgets.Checkbox(False, description = \"Show optional inputs\",indent=False)\n",
+    "        self.toggle_optional.observe(self.toggle)\n",
+    "        self.vb = widgets.VBox(\n",
+    "            children = [self.toggle_optional,self.label,self.warning,self.access_drop_down,self.cloning_drop_down,self.input_description,self.input_name,self.button,self.output], \n",
+    "            layout=wu.vbox_layout\n",
```

Comment:
Nit: Lets be consistent with whitespacing around the equal sign

Reply:
fixed!